classes/autoptimizeToolbar.php

```diff
@@ -0,0 +1,127 @@
+<?php
+if ( ! defined( 'ABSPATH' ) ) exit; // Exit if accessed directly
+
+class autoptimizeToolbar {
+
+	public function __construct()
+	{
+		// If Cache is not available we don't add the Autoptimize Toolbar
+		if( !autoptimizeCache::cacheavail() ) return;
+
+		// Load admin toolbar feature once WordPress, all plugins, and the theme are fully loaded and instantiated.
+		add_action( 'wp_loaded', array( $this, 'load_toolbar' ) );
+	}
+
+	public function load_toolbar()
+	{
+		// We check that the current user has the appropriate permissions
+		if( current_user_can( 'manage_options' ) )
```

Comment:
Thanks for the Tip.
Later, in the next pull-request, you would like to incorporate some options in the plugin configuration page to make changes to these conditions through check boxes? ... as well as the ability to modify the maximum size of the cache... etc.. etc ?